### ISSUE TITLE
SSCBench-KITTI-360 Velodyne

### ISSUE THREAD

**Issue by Weihua-Wang** [pkqbajng/CGFormer#11]
Thanks for sharing this interesting work!
I would like to know where the velodyne data for your KITTI 360 was downloaded from, the data I downloaded from this link doesn't seem right, [https://www.cvlibs.net/datasets/kitti-360/download.php](url) .

**Comment by pkqbajng** [comment]
> Thanks for sharing this interesting work! I would like to know where the velodyne data for your KITTI 360 was downloaded from, the data I downloaded from this link doesn't seem right, [https://www.cvlibs.net/datasets/kitti-360/download.php](url) .
Yeah, it is downloaded from this [link](https://www.cvlibs.net/datasets/kitti-360/download.php). But I merge the data with the raw camera images manully. Simply copying the data and organizing them as the guidance would be ok.

**Comment by Weihua-Wang** [comment]
Thank you for your response. We downloaded the data from the provided link, but we found that the images in the sequence "2013_05_28_drive_0002_sync" do not align with the Velodyne sequence. Could you please provide detailed information on how you prepared this dataset? Your answer would be very helpful for our work. Thank you.

**Comment by pkqbajng** [comment]
> Thank you for your response. We downloaded the data from the provided link, but we found that the images in the sequence "2013_05_28_drive_0002_sync" do not align with the Velodyne sequence. Could you please provide detailed information on how you prepared this dataset? Your answer would be very helpful for our work. Thank you.

I just download the data and move the files. Could you please provide a screenshot for your problem?

**Comment by pkqbajng** [comment]
> Thank you for your response. We downloaded the data from the provided link, but we found that the images in the sequence "2013_05_28_drive_0002_sync" do not align with the Velodyne sequence. Could you please provide detailed information on how you prepared this dataset? Your answer would be very helpful for our work. Thank you.

![image](https://github.com/user-attachments/assets/9864d53b-cdaa-433c-b5a9-1cb5e106b370)
My data shares the same length. There seems no problem.

**Comment by Weihua-Wang** [comment]
<img width="515" alt="7912_1730458270_hd" src="https://github.com/user-attachments/assets/201991df-bb90-40f2-a62f-4d1167a01549">
This is the length of the original data.
<img width="530" alt="7911_1730458269_hd" src="https://github.com/user-attachments/assets/8595046e-ae0d-4ec4-836d-e9ac2e606aae">
<img width="537" alt="7913_1730458270_hd" src="https://github.com/user-attachments/assets/4f3801b3-5bc3-4346-9444-19c792848f52">
The starting numbers of the two sequences are different, and their lengths vary as well. How can I select a portion of the Velodyne data to ensure that its count aligns with the image data?

**Comment by pkqbajng** [comment]
> <img alt="7912_1730458270_hd" width="515" src="https://private-user-images.githubusercontent.com/116944457/382260827-201991df-bb90-40f2-a62f-4d1167a01549.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzA0NjE3MTksIm5iZiI6MTczMDQ2MTQxOSwicGF0aCI6Ii8xMTY5NDQ0NTcvMzgyMjYwODI3LTIwMTk5MWRmLWJiOTAtNDBmMi1hNjJmLTRkMTE2N2EwMTU0OS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTAxJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTEwMVQxMTQzMzlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT02YmIwMmQzNDhiOGEwYjA3ZWRjNDlhODdjZTIzNjU1MGJkYjQ2ZjBlZGM2NDAzNDBhNmQ3ZWVhY2Q5ZmQyYTgwJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.8HloNMqIwlM0eemBkQRWV9u7gKvjwfg_4lvLtUPLIog">
> This is the length of the original data.
> 
> <img alt="7911_1730458269_hd" width="530" src="https://private-user-images.githubusercontent.com/116944457/382261827-8595046e-ae0d-4ec4-836d-e9ac2e606aae.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzA0NjE3MTksIm5iZiI6MTczMDQ2MTQxOSwicGF0aCI6Ii8xMTY5NDQ0NTcvMzgyMjYxODI3LTg1OTUwNDZlLWFlMGQtNGVjNC04MzZkLWU5YWMyZTYwNmFhZS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTAxJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTEwMVQxMTQzMzlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1hYzFiMmVkZDQ5NTA2NTVjMGE4NzFlZjFjNWUzZWMzZmFmY2FmYzhhNjY1NWM5YmQzN2EwNjc4ZmY3NmVmYTIyJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.z4zYzvztreqQzSK29b8tvXy3INx-0_wRayhNL7xRs8s"> <img alt="7913_1730458270_hd" width="537" src="https://private-user-images.githubusercontent.com/116944457/382261877-4f3801b3-5bc3-4346-9444-19c792848f52.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzA0NjE3MTksIm5iZiI6MTczMDQ2MTQxOSwicGF0aCI6Ii8xMTY5NDQ0NTcvMzgyMjYxODc3LTRmMzgwMWIzLTViYzMtNDM0Ni05NDQ0LTE5Yzc5Mjg0OGY1Mi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTAxJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTEwMVQxMTQzMzlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1mZmI2YmU1MGMxZGI1OWUxODhiODllMmIyM2E5M2M5OTZkZjkwMjcyNWIyNTdlZTFiZDEzM2FmNzEzZDQ0MmJkJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.AqKhC4R6_mpWXM2CnpMUWT3akztn6TBUGVDGmilOJlA">
> The starting numbers of the two sequences are different, and their lengths vary as well. How can I select a portion of the Velodyne data to ensure that its count aligns with the image data?

Hi, did you download data from the SSCBench? Here is the link for [SSCBench-KITTI-360](https://huggingface.co/datasets/ai4ce/SSCBench/tree/main/sscbench-kitti). The data may contain lidar data. As I didn't perform any operation on the source data and I forgot the detailed process. Sorry for causing your misunderstanding.

Also, I found that the serial number of the  lidar are different. The following image is mine. Maybe downloading the data using the above link from the SSCBench-KITTI-360 can solve this problem.
![image](https://github.com/user-attachments/assets/ce032559-e362-4244-8ef3-abbe9c05838a)

**Comment by Weihua-Wang** [comment]
Thanks，You are right, the problem is solved.
